scipy/spatial/transform/tests/test_rotation.py

```diff
@@ -1897,6 +2075,7 @@ def test_slerp_decreasing_times(xp):
             Slerp(t, r)
 
 
+@make_xp_test_case(Slerp.__call__)
```

Comment:
This does not test jax.jit nor dask graph materialization because you did not put `Slerp` among `lazy_xp_modules`